src/Models/ApplicantModel.php

```diff
@@ -136,11 +139,19 @@ public function getApplicants(
         $query->bindValue(':name', $name);
         $query->bindValue(':stageId', $stageId);
         $query->bindValue(':cohortId', $cohortId);
+        $query->bindValue(':cohortId2', $cohortId);
         $query->bindValue(':offsets', $offset, \PDO::PARAM_INT);
         $query->bindValue(':numberPerPage', $this->numberPerPage, \PDO::PARAM_INT);
         $query->execute();
         $applicants = $query->fetchAll();
-        foreach ($applicants as $applicant) {
+        foreach ($applicants as $key => $applicant) {
+            if (
+                !empty($applicant->getChosenCourseId()) && $cohortId !== '%' &&
+                $applicant->getChosenCourseId() !== $cohortId
+            ) {
+                unset($applicants[$key]);
```

Comment:
can you add a comment above this line that explains what this is doing. this code is not very clear so needs an explanation why this is being done